crates/pop-contracts/src/test.rs

```diff
@@ -31,26 +38,39 @@ pub fn test_e2e_smart_contract(path: &Option<PathBuf>) -> Result<(), Error> {
 	Ok(())
 }
 
-#[cfg(feature = "unit_contract")]
 #[cfg(test)]
 mod tests {
 	use super::*;
-	use std::fs;
 	use tempfile;
 
-	fn setup_test_environment() -> Result<tempfile::TempDir, Error> {
+	#[test]
+	fn test_contract_test() -> Result<(), Error> {
```

Comment:
These tests only check whether it works correctly in the positive scenario. I'm missing tests where scenarios that should not work are tested as well. For example, I am now able to run `pop test contract` in my parachain crate.